config/routes.rb

```diff
@@ -3,9 +3,12 @@
   get '/login', to: 'session#new'
   post '/login', to: 'session#create'
   delete '/logout', to: 'session#destroy'
-  resources :users, only: [:new, :index, :create, :show]
+  get '/dashboard', to: 'users#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.